cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -50,7 +50,6 @@ def db_restore(temp_appliance_extended_db):
         ssh.get_file("/home/backups/otsuman_db_bottlenecks/REGION{}".format(ver), region_filename)
         guid_filename = "/tmp/GUID_{}".format(fauxfactory.gen_alphanumeric())
         ssh.get_file("/home/backups/otsuman_db_bottlenecks/GUID{}".format(ver), guid_filename)
-    db_storage.close()
```

Comment:
I like this change, I think as part of it though, we rename it ```db_storage``` to ```db_storage_ssh``` to make it clearer.